fuzz/compile-and-execute/src/main.rs

```diff
@@ -0,0 +1,185 @@
+#![no_main]
+
+use rand::prelude::*;
```

Comment:
this file doesn't look like it's in the normal place where `cargo fuzz` places fuzz targets.. is that on purpose?

Reply:
that's from the tutorial.  the honest truth is im learning this.  i really don't know what else to say.

Reply:
If you run:
```
cargo fuzz add <name-of-target>
```

It will update `Cargo.toml` and add an empty `.rs` file you can fill in